### DISCUSSION TITLE
Next.js & Django Project Successfully Built but Encountering 404 Error

### DISCUSSION THREAD

**Discussion by ammarhairiHD** [vercel/next.js#69015] [Help]
### Summary

 I successfully built and deployed my project, which uses Next.js for the frontend and Django for the backend, using Vercel. Despite the build process completing without major issues, I am encountering a 404 error on the deployed site. I have moved all the files to the app/ directory, and the build logs show that the project was compiled successfully. However, there is an error related to fetching data from localhost:8000, which is not accessible in the Vercel build environment.

### Additional information

```js
✓ Compiled successfully
Linting and checking validity of types ...
Collecting page data ...
Error fetching beta testers: TypeError: fetch failed
    at node:internal/deps/undici/undici:13178:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async globalThis.fetch (/vercel/path0/young_guards/.next/server/chunks/638.js:1:36415)
    at async Object.l [as generateStaticParams] (/vercel/path0/young_guards/.next/server/app/(beta-tester)/beta-tester/[securepass]/page.js:1:4339)
    at async buildParams (/vercel/path0/young_guards/node_modules/next/dist/build/utils.js:1025:40)
    at async /vercel/path0/young_guards/node_modules/next/dist/build/utils.js:1042:33
    at async /vercel/path0/young_guards/node_modules/next/dist/build/utils.js:1172:114

The frontend folder is named young_guards, which is the root directory for Next.js.
```

### Example

https://yg-virid.vercel.app/

**Comment by dfout** [comment]
I am having the same issue currently
